kernel/Kconfig

```diff
@@ -557,6 +557,8 @@ config SYS_CLOCK_TICKS_PER_SEC
 	  granularity, not necessarily the number or frequency of
 	  interrupts delivered to the kernel.
 
+	  A value of 0 completely disables timer support in the kernel.
```

Comment:
How this works with the current code?

Example:
```
#if defined(CONFIG_SYS_CLOCK_EXISTS) && \
        (CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC == 0)
#error "SYS_CLOCK_HW_CYCLES_PER_SEC must be non-zero!"
#endif
```

Reply:
I assume this is in a file that doesn't even get compiled in that case, so this is effectively just a compile-time check that the Kconfig deps are correct